pkg-r/R/chat.R

```diff
@@ -380,8 +380,16 @@ chat_append_message <- function(
     ui <- process_ui(content, session)
   }
 
+  msg_content <- ui[["html"]]
+  if (is_html) {
+    msg_content <- sprintf(
+      "\n\n````````{=html}\n%s\n````````\n\n",
```

Comment:
Pairs with the custom raw-html block in the markdown parser, but with extra backticks to make sure it isn't accidentally closed by internal content.

Reply:
Let's leave a comment in the source about this?